### PR TITLE
Fix linter warnings for forced unwrapping

### PR DESCRIPTION
**Pull request by rodericj**

### CODE REVIEW

**Comment by rodericj** [comment]
Thanks for the fast 👍

File: SayTheirNames/Source/Components/MainTabBarController.swift [hunk 1]
```diff
@@ -64,7 +64,10 @@ class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
         tabBar.barTintColor = defaultBarTint
         tabBar.tintColor = defaultTint
         tabBar.unselectedItemTintColor = defaultUnselectedTint
-        UITabBarItem.appearance().setTitleTextAttributes([NSAttributedString.Key.font: UIFont(name: "Karla-Regular", size: 11)!], for: .normal)
+        guard let karlaFont = UIFont(name: "Karla-Regular", size: 11) else {
```
Comment (Kilo-Loco) [inline SayTheirNames/Source/Components/MainTabBarController.swift:67]: We should try to use our semantic fonts when possible to keep consistency
```suggestion
        let karlaFont = UIFont.STN.tabButtonTitle
```

Comment (rodericj) [reply SayTheirNames/Source/Components/MainTabBarController.swift:67]: Done: https://github.com/Say-Their-Name/say-their-names-ios/pull/55

Comment (rodericj) [reply SayTheirNames/Source/Components/MainTabBarController.swift:67]: Done in #55

**Pull request merged**